pkg/flow/logs.go

```diff
@@ -354,7 +354,12 @@ func (srv *server) storeLogMsg(l *logMessage) error {
 		return fmt.Errorf("starting a transaction: %w", err)
 	}
 	defer rollback(tx)
-	lc := tx.LogMsg.Create().SetMsg(l.msg).SetT(l.t)
+	t := ""
```

Comment:
Minor nitpick: I'd prefer if this variable was named `tag`, or even `x`. `t` is confusing because it's a common abbreviation for time, and time is relevant here as you can see a few lines down where we have `SetT(l.t)`.